### DISCUSSION TITLE
Dragging causes elements in panel to re-render

### DISCUSSION THREAD

**Discussion by nmoagit** [bvaughn/react-resizable-panels#331] [Q&A] [answered]
Hi, I have an issue where whenever a `PanelResizeHandle` is clicked it causes the contents of the panel to re-render. Is there a way to prevent this?

**Comment by bvaughn** [comment, marked as answer]
Memoization:
https://react.dev/reference/react/memo

There isn’t really much to go on here, so I’m going to close this issue. If you’d like more help, please provide more information.
